upi/openstack/inventory.yaml

```diff
@@ -0,0 +1,20 @@
+all:
+  hosts:
+    localhost:
+      ansible_connection: local
+      ansible_python_interpreter: "{{ansible_playbook_python}}"
+
+      os_subnet_range: '192.0.2.0/24'
```

Comment:
This IP range is only supposed to be used for examples and documentation and the RFC encourages routers etc. to drop any traffic going through it. This is fine for the UPI doc but not as a default config for these playbooks.

I think we should either always require the caller to set this (no default -- and then we can mention the range above in the docs) or use one that should work (barring IP conflicts).

The default IPI MachineCIDR range is `10.0.0.0/16` -- I think we can just use the same here.